MISC/N_0xa09f896ce912481f.md

```diff
@@ -10,6 +10,11 @@ int _0xA09F896CE912481F(BOOL p0);
 
 ```
 GET_S*
+
+Something to do with Singleplayer DLC
+Only occurances in startup.c and sp_dlc_registration.c
+
+Probably dead at this point since SP DLC is out of the question
```

Comment:
Seems opinionated. I'd keep this out, or clarify that it's a singleplayer function.